asyncio_mqtt/error.py

```diff
@@ -31,10 +31,8 @@ def __init__(self, rc: int | mqtt.ReasonCodes):
         if isinstance(rc, mqtt.ReasonCodes):
             return super().__init__(rc)
         msg = "Connection refused"
-        try:
+        if rc in _CONNECT_RC_STRINGS:
             msg += f": {_CONNECT_RC_STRINGS[rc]}"
-        except KeyError:
-            pass
         super().__init__(rc, msg)
```

Comment:
The pythonic way is to ask for forgiveness, not permission. In other languages (like C++), it's the other way around.
In any case, let's keep the (admittedly verbose) `try..except` block to stay pythonic.

See https://stackoverflow.com/questions/12265451/ask-forgiveness-not-permission-explain

Reply:
It's a bit nitpicky on my side, I'll admit that. 😄 You decide which version that you want to keep here.